corehq/messaging/smsbackends/vertex/models.py

```diff
@@ -65,7 +65,17 @@ def populate_params(self, msg_obj):
         }
         return params
 
+    def phone_number_is_valid(self, phone_number):
+        phone_number = strip_plus(phone_number)
+        # Phone number must be an Indian phone number
+        # Also avoid processing numbers that are obviously too short
+        return phone_number.startswith('91') and len(phone_number) > 3
```

Comment:
the check for `91` seems a good add. I am not sure if length of 4 digits around will happen.. but a length of 9 is possible considering someone mistypes or skips a digit.